dingo_viz/package.xml

```diff
@@ -14,6 +14,8 @@
   <exec_depend>joint_state_publisher_gui</exec_depend>
   <exec_depend>rviz</exec_depend>
   <test_depend>roslaunch</test_depend>
+  <exec_depend>rqt_robot_monitor</exec_depend>
```

Comment:
Small nitpick: can you alphabetize the exec_depends, and put the test_depends in a separate section after?